xla/tools/multihost_hlo_runner/hlo_runner_main.cc

```diff
@@ -163,9 +164,24 @@ RunningOptionsFromFlags(const HloRunnerConfig& opts) {
   FunctionalHloRunner::RunningOptions out;
   TF_ASSIGN_OR_RETURN(out.module_argument_mode,
                       ArgumentModeFromString(opts.hlo_argument_mode));
+  // add parser for output mode (from functional_hlo_runner.h::AbslParseFlag)
```

Comment:
Why not just call AbslParseFlag?

If there is a good reason, I'd remove this comment, since it doesn't add much value.